### PR TITLE
Fixes #1411 use nullable

### LINKED ISSUE #1411: Use nullable color binder

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/OSPSuite.Assets/UIConstants.cs [hunk 1]
```diff
@@ -194,7 +194,7 @@ public static class Captions
       public static readonly string EditManually = "Edit manually";
       public static readonly string ShouldColorGroupObservedData = "Color group observed data from same folder when dropping to chart";
       public static readonly string ColorGroupObservedDataContextMenu = "Color group when adding to chart";
-      public static readonly string EditAllCurvesProperties = "Edit options for selected";
+      public static readonly string EditAllCurvesProperties = "Edit Options for Selected";
```
Comment (msevestre) [inline src/OSPSuite.Assets/UIConstants.cs:197]: This was not following the other context menu. Pascal case expcet for binding words such as to, at, for etc..

File: src/OSPSuite.UI/Views/Charts/CurveMultiItemEditorView.cs [hunk 2]
```diff
@@ -40,8 +40,7 @@ public override void InitializeBinding()
          base.InitializeBinding();
 
          _screenBinder.Bind(x => x.Color)
-            .To(colorPickEdit1)
-            .OnValueUpdated += colorChanged;
+            .To(colorPickEdit);
```
Comment (msevestre) [inline src/OSPSuite.UI/Views/Charts/CurveMultiItemEditorView.cs:43]: Always rename the default control name to remove the 1 generated by default. I missed it previously

File: src/OSPSuite.UI/Views/Charts/CurveSettingsView.cs [hunk 3]
```diff
@@ -389,10 +389,11 @@ private void onPopupMenuShowing(object sender, PopupMenuShowingEventArgs e)
          if (gridViewMenu == null)
             return;
 
-         if (gridView.GetSelectedRows().Length == 0)
+         //only one or no curve selected. We do not show the option
+         if (gridView.GetSelectedRows().Length <= 1)
```
Comment (msevestre) [inline src/OSPSuite.UI/Views/Charts/CurveSettingsView.cs:393]: @georgeDaskalakis This was not working wiht only 1 selected. (e.g. the view was not showing up). Also I don't think it makes sense with only one selected item

File: src/OSPSuite.UI/Views/Charts/CurveSettingsView.cs [hunk 4]
```diff
@@ -389,10 +389,11 @@ private void onPopupMenuShowing(object sender, PopupMenuShowingEventArgs e)
...
             return;
 
-         var copyAllMenu = new DXMenuItem(Captions.EditAllCurvesProperties, (o, args) => onEditProperties());
+         var copyAllMenu = new DXMenuItem(Captions.EditAllCurvesProperties, (o, args) => onEditProperties(), ApplicationIcons.Edit);
```
Comment (msevestre) [inline src/OSPSuite.UI/Views/Charts/CurveSettingsView.cs:396]: Icon feels better

**Comment by georgeDaskalakis** [comment]
@msevestre looks good Michael! I am going to merge

**Pull request merged**